site/content/index.md

```diff
@@ -57,6 +57,27 @@ See [mage homebrew formula](https://formulae.brew.sh/formula/mage).
 
 See [scoop](https://scoop.sh/).
 
+**Using asdf**
+
+The [asdf version manager](https://asdf-vm.com/) is a tool for installing release binaries from Github. With asdf installed, the [asdf plugin for mage](https://github.com/mathew-fleisch/asdf-mage) can be used to install any released version of mage.
+
+```shell
+# Install asdf plugin for mage
+asdf plugin add mage
+
+# Show all installable versions
+asdf list-all mage
+
+# Install specific version
+asdf install mage latest
+
+# Set a version globally (on your ~/.tool-versions file)
+asdf global mage latest
+
+# Now mage commands are available
+mage --version
+```
```

Comment:
since this is a specific installer that not everyone will use, let's try to keep the install sections short.  Can you make this a lot shorter, following the examples of the other installers above?  Like a link to the docs, and then just the 2-3 lines you need to run to install?

Reply:
how's that?

Reply:
looks good!